ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/utility/FileUtility.java

```diff
@@ -12,6 +12,18 @@
 
 public final class FileUtility {
     public static final FileChooser.ExtensionFilter IMAGE_EXTENSION_FILTER = new FileChooser.ExtensionFilter("PNG", "*.png");
+    public static final FileChooser.ExtensionFilter HISTORY_EXTENSION_FILTER = new FileChooser.ExtensionFilter("Saved Chess Game", "*.chesscrafter");
+    public static final FileChooser.ExtensionFilter CONFIG_EXTENSION_FILTER = new FileChooser.ExtensionFilter("Crafted Game", "*.craftconfig");
+    public static final FileChooser.ExtensionFilter PIECE_EXTENSION_FILTER = new FileChooser.ExtensionFilter("Piece", "*.piece");
+
+    private static final String HIDDEN_DIR;
+    private static final String IMAGES = "images";
+    private static final String VARIANTS = "variants";
+    private static final String PIECES = "pieces";
+    //private static final String GAMES_IN_PROGRESS = "gamesInProgress";
+    private static final String SAVED_GAMES = "Saved Games";
```

Comment:
Let's avoid directory names with spaces, since it can cause issues on some platforms